### DISCUSSION TITLE
How to use text objects from multiple languages ?

### DISCUSSION THREAD

**Discussion by uros-5** [helix-editor/helix#9163] [Q&A] [answered]
Here is quick example:
```html
<p> hello world</p>
<script>
  function example() {
    console.log("hello world");
 }
</script>
```

Why `]f` or `[f` doesn't move cursor to `example` function ?

How to combine text objects from JavaScript ?

**Comment by gabydd** [comment, marked as answer]
That's currently not possible but will be fixed by https://github.com/helix-editor/helix/pull/5176
